backend/src/test/java/de/bund/digitalservice/ris/norms/adapter/input/restapi/controller/ProprietaryControllerTest.java

```diff
@@ -266,4 +266,91 @@ void itReturnsNotFoundIfNormIsNotFound() throws Exception {
           .andExpect(status().isNotFound());
     }
   }
+
+  @Nested
+  class getProprietaryEinzelelementAtDate {
+    @Test
+    void returns404IfNormNotFound() throws Exception {
```

Comment:
I just saw now this test. Meaning we can't remove the try/catch of the controller. It would be great if we can merge this branch before Sebastian is done, so that he can remove the try/catch there

Reply:
Ok for me